src/sentry/replays/lib/query.py

```diff
@@ -152,8 +155,20 @@ def _has_condition(
         if isinstance(value, list):
             return self._has_any_condition(Op.IN if operator == Op.EQ else Op.NOT_IN, value)
 
+        if self.is_uuid:
+            # Client side UUID validation.  If this fails we use a condition which is always
+            # false.  E.g. 1 == 2.  We don't use toUUIDOrZero because our Clickhouse version does
+            # not support it.
```

Comment:
why not just raise an error here?

Reply:
@JoshFerge UX.  Do you have a preference between a 400 error and a falsey query condition?

Reply:
hrmmm, good point. with the search bar 400s aren't a great experience. yeah this is good!